spec/models/course_spec.rb

```diff
@@ -284,6 +284,13 @@
         it "does not persist the course" do
           expect(course).not_to be_persisted
         end
+
+        xcontext "Which is then saved" do
```

Comment:
Should this be left pending?

Reply:
I left it pending for now because I was wondering if it was worth attempting to create some kind of clean up after creation to remove the site_statuses, but after my quick spike I didn't get anywhere so :man_shrugging: 

I can remove it for now, but I do have concerns about potential risks when doing distinct if they're still duplicated. Though I do want to get this out the door :grimacing:

Reply:
I might spend another 30 minutes on it prior to merging and see

Reply:
This taken up tons of cycles already. At the risk of spawning more bugs, maybe we should just get it in and worry about the fallout later. :-/